java/arcs/core/storage/CapabilitiesResolver.kt

```diff
@@ -127,6 +127,8 @@ class CapabilitiesResolver(
 
     /** Returns list of creator info corresponding to the given [Capabilities]. */
     /* internal */ fun findCreators(capabilities: Capabilities): List<StorageKeyCreatorInfo> {
+        var result = creators.filter { creator -> creator.capabilities.equals(capabilities) }
```

Comment:
this is a bit brittle, may we could introduce ranking between creators, ie if two creators both can support this list of of capabilities, choose based on ...

Reply:
i agree, and i'm going to address it properly in the redesigned store capabilities implementation (one of the reasons we needed the redesign), here i just wanted a quick workaround to enable sqlite inmemory for TTL support.

Reply:
Nit: you can use `==` instead of `.equals` in kotlin :)